lib/koala/api/graph_error_checker.rb

```diff
@@ -15,6 +15,9 @@ def initialize(http_status, body, headers)
       # token.
       AUTHENTICATION_ERROR_CODES = [102, 190, 450, 452, 2500]
 
+      # Partial condition for an error to be indentified as permissions error
+      PERMISSION_ERROR_CODES = [100, 200]
```

Comment:
```suggestion
      PERMISSION_ERROR_CODES = [100, 200..299]
```
Could you add the full range for 2xx errors. maybe the check will have to be modified to fit the range in the array